HandleNHNotificationCallActivity/handler.ts

```diff
@@ -98,9 +102,22 @@ export const getCallNHServiceActivityHandler = (
             retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
           );
         case NotifyKind.Notify:
-          return notify(message.installationId, message.payload).mapLeft(e =>
-            retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
-          );
+          return notify(message.installationId, message.payload)
+            .mapLeft(e =>
+              retryActivity(context, `${logPrefix}|ERROR=${toString(e)}`)
+            )
+            .chainFirst(
+              taskEither.of(
+                telemetryClient.trackEvent({
+                  name: "api.messages.notification.push",
```

Comment:
```suggestion
                  name: "api.messages.notification.push.sent",
```

We will send "api.messages.notification.push.queued" in io-functions-services